app/views/users/index.html.erb

```diff
@@ -1,2 +1,44 @@
-<h1>Users#index</h1>
-<p>Find me in app/views/users/index.html.erb</p>
+
```

Comment:
- [ ] Your root is users#index, meaning this view should display when the server is started. From the screeshots below, nothing displays in the browser.
- Terminal
![image](https://user-images.githubusercontent.com/62849872/216497878-9a39fc07-6906-47de-baff-d817362d4533.png)
- Browser is white
![image](https://user-images.githubusercontent.com/62849872/216498079-9cada0f2-5177-4770-85ae-7f76be108566.png)

Kindly check this and ensure it is an issue to be fixed